src/nrtk_explorer/app/ui/image_list.py

```diff
@@ -210,22 +279,40 @@ def __init__(self, **kwargs):
                 rows=(
                     r"""image_list_ids.map((id) =>
                             {
-                                const meta = get(`meta_${id}`)?.value ?? {original_ground_to_original_detection_score: 0, ground_truth_to_transformed_detection_score: 0, original_detection_to_transformed_detection_score: 0}
                                 const original_id = `img_${id}`
                                 const transformed_id = `transformed_img_${id}`
+
+                                const originalAnnotations = Object.entries(inference_models_obj).reduce(function(acc, [model_id, model]){
+                                    acc[model_id] = get(`result_${original_id}_${model.name}`);
+                                    return acc;
+                                }, {});
+
+                                const transformedAnnotations = Object.entries(inference_models_obj).reduce(function(acc, [model_id, model]){
+                                    acc[model_id] = get(`result_${transformed_id}_${model.name}`);
+                                    return acc;
+                                }, {});
+
+                                const originalScores = Object.entries(inference_models_obj).reduce(function(acc, [model_id, model]){
+                                    acc[model_id] = get(`score_${original_id}_${model.name}`);
+                                    return acc;
+                                }, {});
+
+                                const transformedScores = Object.entries(inference_models_obj).reduce(function(acc, [model_id, model]){
+                                    acc[model_id] = get(`score_${transformed_id}_${model.name}`);
+                                    return acc;
+                                }, {});
+
```

Comment:
keep in mind that you can create JS file that you can load so you could write something like `utils.nrtk.filterImagesByXYZ(image_list_ids)`